### PR TITLE
fix: #211

### PR DESCRIPTION
**Pull request by aaronsteers**
## Checklist

- [ ] I have run this code and it appears to resolve the stated issue
- [ ] `README.md` updated and added information about my change

### CODE REVIEW

File: src/manifest/dbtWorkspaceFolder.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ export class DBTWorkspaceFolder implements Disposable {
         if (j !== i) {
           if (projectFiles[i].fsPath.startsWith(projectFiles[j].fsPath)) {
             window.showWarningMessage(
-              `dbt Power User detected a project located in ${projectFiles[j]} that is contained inside another project ${projectFiles[j]}. This is an unsupported configuration. If you believe this is a valid configuration, please open a github issue.`
+              `dbt Power User detected a project located in ${projectFiles[i]} that is contained inside another project ${projectFiles[j]}. This is an unsupported configuration. If you believe this is a valid configuration, please open a github issue.`
```
Comment (mdesmet) [inline src/manifest/dbtWorkspaceFolder.ts:61]: Please also change both paths to use `.fsPath`, that path will be more meaningful to users.

Comment (mdesmet) [inline src/manifest/dbtWorkspaceFolder.ts:61]:
```suggestion
              `dbt Power User detected a project located in ${projectFiles[i].fsPath} that is contained inside another project ${projectFiles[j].fsPath}. This is an unsupported configuration. If you believe this is a valid configuration, please open a github issue.`
```

**Comment by aaronsteers** [comment]
Thanks, @mdesmet !!

**Comment by mdesmet** [comment]
You're welcome! Let me know how it goes with the multi root testing

**Pull request merged**